docs/getting_started/installation.rst

```diff
@@ -123,22 +106,21 @@ configuration. The admin interface is using SonataBlockBundle to put everything
 in blocks. You just have to tell the block bundle about the existence of the
 admin block:
 
-.. code-block:: yaml
-
-    # config/packages/sonata.yaml
-    sonata_block:
-        default_contexts: [] # this line can be removed for sonata-project/block-bundle >= 3.10.0
```

Comment:
If the docs are for 3.x of Sonata, this can be safely removed because block-bundle is forced to be 3.11